third_party/terraform/website/docs/r/bigtable_gc_policy.html.markdown

```diff
@@ -29,16 +29,16 @@ resource "google_bigtable_instance" "instance" {
 
 resource "google_bigtable_table" "table" {
   name          = "tf-table"
-  instance_name = google_bigtable_instance.instance.name
```

Comment:
Have you tested this resource? I"m not confident that the generated test will succeed.

Reply:
Yes tested locally, it works only for `name` attribute at this point. To support for `id` attribute an issue has been raised. Once that's implemented i will pick up this PR